content/client-lib-development-guide/features.textile

```diff
@@ -69,7 +69,7 @@ h3(#restclient). RestClient
 * @(RSC6)@ @RestClient#stats@ function:
 ** @(RSC6a)@ Returns a @PaginatedResult@ page containing @Stats@ objects in the @PaginatedResult#items@ attribute returned from the stats request
 ** @(RSC6b)@ Supports the following params:
-*** @(RSC6b1)@ @start@ and @end@ are timestamp fields represented as milliseconds since epoch, or where suitable to the language, Date or Time objects. @start@ must be equal to or less than @end@ and is unaffected by the request direction
+*** @(RSC6b1)@ @start@ and @end@ are optional timestamp fields represented as milliseconds since epoch, or where suitable to the language, Date or Time objects. @start@, if provided, must be equal to or less than @end@ if provided or to the current time otherwise, and is unaffected by the request direction
```

Comment:
The use of "if provided" suggests they are optional, thus I would expect the types to be `Time?`. Does this need a bit more clarity perhaps?

Reply:
For what I've seen, `api-default` implicitly makes the type optional. We can make that explicit, but then we should fix the other occurrences.

Reply:
We use `Time` types when they exist and are idiomatic in the language.  Javascript is one of those languages that idiomatically uses Ints to represent time.  Most other languages have common time type attributes, which makes it far easier for developers to consume our API.